kafka/content.md

```diff
@@ -0,0 +1,305 @@
+# What is Apache Kafka®?
+
+Apache Kafka is an open-source event streaming platform used to collect, process, store, and integrate data at scale in real time. It powers numerous use cases including stream processing, data integration, and pub/sub messaging.
+
+Kafka was originally developed at LinkedIn, was open sourced in 2011, and became an Apache Software Foundation project in 2012. It is used by thousands of organizations globally to power mission-critical real-time applications, from stock exchanges, to e-commerce applications, to IoT monitoring &amp; analytics, to name a few.
+
+%%LOGO%%
+
+## Quick start
+
+Start a Kafka broker:
+
+```console
+docker run -d --name broker kafka:latest
+```
+
+Open a shell in the broker container:
+
+```console
+docker exec --workdir /opt/kafka/bin/ -it broker sh
+```
+
+A *topic* is a logical grouping of events in Kafka. From inside the container, create a topic called `test-topic`:
+
+```console
+./kafka-topics.sh --bootstrap-server localhost:9092 --create --topic test-topic
+```
+
+Write two string events into the `test-topic` topic using the console producer that ships with Kafka:
+
+```console
+	./kafka-console-producer.sh --bootstrap-server localhost:9092 --topic test-topic
```

Comment:
If you wrap in triple backticks you should not also indent.

Reply:
Ahh makes sense, changed this for all ``` consoles. Thanks @whalelines !